src/domain/event/utils.ts

```diff
@@ -131,3 +134,20 @@ export const isFutureEvent = (eventData: EventQuery | undefined) =>
 
 export const isEditableEvent = (eventData: EventQuery | undefined) =>
   isFutureEvent(eventData);
+
+export const hasOccurrences = (event: EventFieldsFragment) => {
+  return Boolean(event.pEvent?.occurrences.edges.length);
+};
+
+export const hasComingOccurrences = (event: EventFieldsFragment) => {
```

Comment:
I think adding boolean return type for this function would be good

Reply:
If you change/refactor this it will tell you if you are not returning boolean anymore